sdk/cosmos/azure-cosmos-test/src/main/java/com/azure/cosmos/test/models/FaultInjectionCondition.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.cosmos.faultinjection;
+package com.azure.cosmos.test.models;
```

Comment:
My 2 cents - I woudl still keep it ina separate package - like com.azure.cosmos.test.models.failureInjection or just com.azure.cosmos.test.failureInjection (and then have models and InjectionHelper all in this package)